Content.Server/GameTicking/Commands/SetGamePresetCommand.cs

```diff
@@ -36,32 +36,39 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
 
             var rounds = 1;
 
-            if (args.Length == 2 && !int.TryParse(args[1], out rounds))
+            if (args.Length >= 2 && !int.TryParse(args[1], out rounds))
             {
                 shell.WriteError(Loc.GetString("set-game-preset-optional-argument-not-integer"));
                 return;
             }
 
-            ticker.SetGamePreset(preset, false, rounds);
-            shell.WriteLine(Loc.GetString("set-game-preset-preset-set-finite", ("preset", preset.ID), ("rounds", rounds.ToString())));
+            GamePresetPrototype? decoy = null;
+
+            if (args.Length == 3 && !ticker.TryFindGamePreset(args[2], out decoy))
+            {
+                shell.WriteError(Loc.GetString("set-game-preset-preset-error", ("preset", args[2])));
+                return;
+            }
+
+            ticker.SetGamePreset(preset, false, decoy, rounds);
+            if (decoy == null)
+                shell.WriteLine(Loc.GetString("set-game-preset-preset-set-finite", ("preset", preset.ID), ("rounds", rounds.ToString())));
+            else
+                shell.WriteLine(Loc.GetString("set-game-preset-preset-set-finite-with-decoy", ("preset", preset.ID), ("rounds", rounds.ToString()), ("decoy", decoy.ID)));
         }
 
         public CompletionResult GetCompletion(IConsoleShell shell, string[] args)
         {
-            if (args.Length == 1)
+            return args.Length switch
             {
-                var gamePresets = _prototype.EnumeratePrototypes<GamePresetPrototype>()
-                    .OrderBy(p => p.ID);
-                var options = new List<string>();
-                foreach (var preset in gamePresets)
-                {
-                    options.Add(preset.ID);
-                    options.AddRange(preset.Alias);
-                }
+                1 => CompletionResult.FromHintOptions(CompletionHelper.PrototypeIDs<GamePresetPrototype>(),
+                "<id>"),
```

Comment:
Please also add a FromHint for the second parameter, so it reads a localised hint (which also needs to be created)

`2 => CompletionResult.FromHint(Loc.GetString("set-game-preset-hint-rounds")),`